static/js/components/dashboard/CourseSubRow.js

```diff
@@ -124,11 +125,21 @@ export default class CourseSubRow extends React.Component {
         />;
       }
     }
+    let status = '';
+    if ( courseRun.status === STATUS_NOT_PASSED ) {
+      status = 'Failed';
+    } else if ( courseRun.status === STATUS_PASSED ) {
+      status = 'Passed';
+    }
     return <div className="course-action">
-      { courseRun.status === STATUS_NOT_PASSED ? 'Failed' : '' }
+      { status }
     </div>;
   }
 
+  courseRunGrade = (courseRun: CourseRun) => (
+    [STATUS_NOT_PASSED, STATUS_PASSED].includes(courseRun.status) ? formatGrade(courseRun.final_grade) : ''
```

Comment:
can you add a helper method `isCompletedCourseRun` that performs this logic (`[STATUS_NOT_PASSED, STATUS_PASSED].includes(courseRun.status)`) and call it here and line 166?